cms/templates/studio_xblock_wrapper.html

```diff
@@ -69,35 +69,42 @@
+                                <button class="btn-default edit-button action-button">
                                     <span class="icon fa fa-pencil" aria-hidden="true"></span>
                                     <span class="action-button-text">${_("Edit")}</span>
-                                </a>
+                                </button>
                             </li>
                             % if can_edit_visibility:
                                 <li class="action-item action-visibility">
-                                    <a href="#" data-tooltip="${_("Visibility Settings")}" class="visibility-button action-button">
+                                    <button data-tooltip="${_("Visibility Settings")}" class="btn-default visibility-button action-button">
                                         <span class="icon fa fa-eye" aria-hidden="true"></span>
                                         <span class="sr">${_("Visibility")}</span>
-                                    </a>
+                                    </button>
                                 </li>
                             % endif
                             % if can_add:
                                 <li class="action-item action-duplicate">
-                                    <a href="#" data-tooltip="${_("Duplicate")}" class="duplicate-button action-button">
+                                    <button data-tooltip="${_("Duplicate")}" class="btn-default duplicate-button action-button">
                                     <span class="icon fa fa-copy" aria-hidden="true"></span>
                                     <span class="sr">${_("Duplicate")}</span>
-                                    </a>
+                                    </button>
+                                </li>
+
+                                <li class="action-item action-move">
+                                    <button data-tooltip="${_("Move")}" class="btn-default move-button action-button">
+                                    <span class="icon fa fa-folder-o" aria-hidden="true"></span>
+                                    <span class="sr">${_("Move")}</span>
+                                    </button>
                                 </li>
                             % endif
                         % endif
                         % if can_add:
                             <!-- If we can add, we can delete. -->
                             <li class="action-item action-delete">
-                                <a href="#" data-tooltip="${_("Delete")}" class="delete-button action-button">
+                                <button data-tooltip="${_("Delete")}" class="btn-default delete-button action-button">
                                 <span class="icon fa fa-trash-o" aria-hidden="true"></span>
                                 <span class="sr">${_("Delete")}</span>
-                                </a>
+                                </button>
```

Comment:
@andy-armstrong @cahrens I have replaced all `<a>` tags with `<button>` but feeling uneasy. Do you see any issues? Any concerns?